www/experiments/imgs-wasteful.inc

```diff
@@ -0,0 +1,73 @@
+<?php
+
+$category = "Quick";
+$imgsTooLarge = $testStepResult->getMetric('imgs-too-large');
+$imgResizeURL = Util::getSetting('imgResizeURL', null);
+
+function fullURL($url)
+{
+    global $requests;
+    if (preg_match('/^(https?\:\/\/)/', $url)) {
```

Comment:
probs better to use https://php.net/parse_url, something like
```php
if (parse_url($url, PHP_URL_SCHEME)) {

Reply:
good tip thanks